core/src/main/java/org/apache/carbondata/core/metadata/encoder/Encoding.java

```diff
@@ -57,10 +62,55 @@ public static Encoding valueOf(int ordinal) {
       return ADAPTIVE_DELTA_INTEGRAL;
     } else if (ordinal == RLE_INTEGRAL.ordinal()) {
       return RLE_INTEGRAL;
+    } else if (ordinal == DIRECT_STRING.ordinal()) {
```

Comment:
DIRECT_STRING ordinal in thrift file is 10.  In this enum it is 11. Does it cause problem for old stores if any one uses this method to get encoding?

Reply:
There will not be any problem as this ordinal is not used during thrift writing or any file persistence